### DISCUSSION TITLE
French translation group

### DISCUSSION THREAD

**Discussion by ManuSquall** [microsoft/Web-Dev-For-Beginners#484] [Translations]
Welcome to the french translation group! 🥳🥳

i created this discussion inspired by the one on [ML-For-Beginners](https://github.com/microsoft/ML-For-Beginners/discussions/186) to discuss about more accurrate translation of this curriculum in french :-)!

**Comment by ManuSquall** [comment]
@MedouneSGB what do you think about change the title translation of the 7-1 lesson "HTML Templates and Routes" by "Modèles HTML et routage"; "routage" seems to be more accurate and adapted for the situation. 👀 ?

Ref:
https://github.com/microsoft/Web-Dev-For-Beginners/pull/458/files
https://github.com/microsoft/Web-Dev-For-Beginners/pull/462/files

@gturpin-dev?

If most are ok, i will make the changes

**Reply by MedouneSGB** [reply]
Hi @ManuSquall,
Yes, I think it's better to translate it as "routage".
It's more explicit

**Comment by jlooper** [comment]
Thanks for syncing up here :) And thank you all for your contributions and for @ManuSquall's leadership!

**Comment by joanaquaicoe2-cmyk** [comment]
I want to learn french please
